### DISCUSSION TITLE
请教一下。如何动态修改节点和线的颜色

### DISCUSSION THREAD

**Discussion by huyongdi** [antvis/X6#1924] [Q&A] [answered]
比如我有rect，也有circle节点等等。点击某个按钮后想修改线边框内容等所有颜色，类似view.setStyle这种。目前我用view.setStyle({},ele)可以改到单个ele的样式，但是加入我页面有10个，怎样批量改呢

**Comment by NewByVector** [comment, marked as answer]
可以参考[文档](https://x6.antv.vision/zh/docs/tutorial/basic/cell#attrs-1)
